hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionFileSystem.java

```diff
@@ -100,7 +100,7 @@
    * @param tableDir {@link Path} to where the table is being stored
    * @param regionInfo {@link RegionInfo} for region
    */
-  HRegionFileSystem(final Configuration conf, final FileSystem fs, final Path tableDir,
+  public HRegionFileSystem(final Configuration conf, final FileSystem fs, final Path tableDir,
```

Comment:
Ouch on making this public and HStore public.  The compaction server is a subset of the RegionServer? Should it be in the RegionServer package?